sdk/nodejs/tests/automation/localWorkspace.spec.ts

```diff
@@ -989,6 +989,46 @@ describe("LocalWorkspace", () => {
         const destroyRes = await stack.destroy({ userAgent, refresh });
         assert.match(destroyRes.stdout, /refreshing/);
     });
+    it(`operations accept configFile option`, async () => {
+        // Setup test data
+        const configPath = upath.joinSafe(__dirname, "data", "yaml", "Pulumi.dev.yaml");
+
+        // Create options objects with configFile
+        const previewOpts = { configFile: configPath };
+        const upOpts = { configFile: configPath };
+        const refreshOpts = { configFile: configPath };
+        const destroyOpts = { configFile: configPath };
```

Comment:
These opts are not really used anywhere (other than the couple of equals below, that don't make a lot of sense)